Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -1980,6 +1993,10 @@ QString qMRMLSegmentEditorWidget::masterVolumeNodeID()const
 void qMRMLSegmentEditorWidget::onSegmentationNodeChanged(vtkMRMLNode* node)
 {
   Q_D(qMRMLSegmentEditorWidget);
+  if (!this->mrmlScene() || this->mrmlScene()->IsBatchProcessing())
```

Comment:
@lassoan Do you think a comment is still necessary here? I'm not sure what I would write that wouldn't just rehash why batch processing state is used (to avoid unnecessary updates).